osu.Game/Overlays/OSD/TrackedSettingToast.cs

```diff
@@ -90,9 +62,6 @@ public TrackedSettingToast(SettingDescription description)
                     break;
             }
 
-            textLine2.Origin = optionCount > 0 ? Anchor.BottomCentre : Anchor.Centre;
```

Comment:
what happened to this?

Reply:
I deleted this intentionnally as it visually doesn't change anything now.

Reply:
Hm, it should though. The vertical alignment would hopefully not change as a result of this PR>

Reply:
I did some verifications via visual tests to ensure it doesn't change anything before deleting it.

Reply:
I'm not sure how that could work when this is specifically changing the alignment of the text line when there are options present...

Reply:
here's how the vertical offset is currently applied for reference:

![image](https://user-images.githubusercontent.com/191335/61124133-1812e880-a4e1-11e9-9e8f-a937c79b60f6.png)
![image](https://user-images.githubusercontent.com/191335/61124150-1f39f680-a4e1-11e9-9794-0b6c2a4e8929.png)

Reply:
looks like it was an attention error of mine when I moved the tracked setting's code to its own class, as it actually influences the display ,my bad